packages/fields/src/types/Relationship/views/CreateItemModal.js

```diff
@@ -60,7 +60,11 @@ class CreateItemModal extends Component {
       return;
     }
 
-    const data = arrayToObject(fields, 'path', field => field.serialize(item));
+    const creatable = fields
+      .filter(({ isPrimaryKey }) => !isPrimaryKey)
```

Comment:
This inadvertently fixes a bug introduced in #1519 where the Id field would still be visible when doing a nested create from an item details page. 🎉